### PR TITLE
Modify Table and JS code

### PR DESCRIPTION
**Pull request by AgisKounelis**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,58 +1,53 @@
-var win = false;
-var difficulty = +document.getElementById("difficulty-level").value;
+var difficulty = document.getElementById("difficulty-level").value;
 var input = document.getElementById("user-input");
 input.maxLength = difficulty;
-let compInput = [];
+var targetNumber = [];
+
```
Comment (jemimaabu) [inline index.js]: The code style uses single line spacing rather than double, for uniformity sake

File: index.js [hunk 2]
```diff
@@ -64,23 +59,30 @@ function guess() {
...
-		if (win) {
-			return;
+		
+		let result = countDeadAndWounded(userInput);
+		
+		if (result.dead == difficulty) {
+			addResultRow(userInput, "All dead, you win!");;
 		} else {
-			addResultRow(userInput, deadAndWounded(compInput, userInput));
+			addResultRow(userInput, "dead: " + result.dead + " | wounded: " + result.wounded);
```
Comment (jemimaabu) [inline index.js]: If the result display text is changed, the instructions should be updated to reflect the change

File: style.css [hunk 3]
```diff
@@ -115,13 +115,19 @@ select {
...
-	overflow-y: auto;
-	height: 50vh;
 	width: 100%;
 	max-width: 700px;
 	margin-left: auto;
 	margin-right: auto;
+
+}
+
+
```
Comment (jemimaabu) [inline style.css]: Double line spacing is not required

File: index.js [hunk 4]
```diff
@@ -64,23 +59,30 @@ function guess() {
...
-		error.innerHTML = "Input cannot be empty"
+	if (userInput.length == 0) {
+		error.innerHTML = "Input cannot be empty";
 	} else if (isNaN(userInput)) {
-		error.innerHTML = "Input must be a number"
-	} else if (userInput.length !== difficulty) {
-		error.innerHTML = `Input must be ${difficulty} characters`
+		error.innerHTML = "Input must be a number";
+	} else if (userInput.length != difficulty) {
+		console.log(userInput.length + " " + difficulty);
```
Comment (jemimaabu) [inline index.js]: Delete console statement

File: index.html [hunk 5]
```diff
@@ -57,14 +57,16 @@ <h3>Rules</h3>
          </div>         
       </div>
       <div id="table-wrapper">
-         <div id="result-table">
-            <table>
+	  <table>
```
Comment (jemimaabu) [inline index.html]: Maintain indentation
```suggestion
	  <table>
            <thead>
               <tr>
                  <th>Guess</th>
                  <th>Result</th>
               </tr>
            </thead>
         </table>
         <div id="scroll-table">
            <table>
               <tbody id="result-body">
               </tbody>
            </table>
         </div>
```

**Comment by AgisKounelis** [comment]
good changes

File: index.html [hunk 6]
```diff
@@ -57,14 +57,16 @@ <h3>Rules</h3>
...
                <thead>
                   <tr>
                      <th>Guess</th>
                      <th>Result</th>
                   </tr>
                </thead>
+			   	  </table>
```
Comment (jemimaabu) [inline index.html]: This is poorly indented

**Pull request merged**